src/private/meetingAudio.ts

```diff
@@ -0,0 +1,31 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+import { SdkError, ErrorCode } from '../public/interfaces';
+
+export namespace meetingAudio {
+  /**
+   * @private
+   * Hide from docs
+   * ------
+   * Allows an app to toggle the incoming audio speaker setting for the meeting user from mute to unmute or vice-versa
+   * @param callback Callback contains 2 parameters, error of type SdkError and result of type boolen or null.
+   * result can either contain the isSpeakerMuted value, incase of a successful toggle or null when the toggling fails
```

Comment:
isSpeakerMuted value has no significance here. "result can either contain the isSpeakerMuted value" this part can be removed

Reply:
Removed it. Thanks